.github/workflows/testdocker.yml

```
@@ -0,0 +1,12 @@
+name: test docker auth
+on:
+    workflow_dispatch:
+jobs:
+    build-and-push-image:
+        runs-on: ubuntu-latest        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)